packages/expo-modules-core/ios/Swift/Logging/LogType.swift

```diff
@@ -5,7 +5,7 @@ import os.log
 /**
  An enum with available log types.
  */
-public enum LogType: Int {
+@objc public enum LogType: Int {
```

Comment:
- `@objc` in the separate line
- If you're exposing Swift code to Objective-C, we should prefix it with `EX` so its usages in Objective-C will be versioned correctly

```suggestion
@objc(EXLogType)
public enum LogType: Int {
```

Reply:
Will remove this change